src/test/mod.rs

```diff
@@ -693,6 +696,12 @@ fn print_mismatches<T: Fn(u32) -> String>(
 }
 
 fn read_config(filename: &Path) -> Config {
+    if filename.file_name().map_or(false, |f| f == "Cargo.toml") {
+        let mut config = Config::default();
+        config.set().format_cargo_toml(true);
+        return config;
+    }
```

Comment:
Instead of always setting this to `true` would it be possible to add `#rustfmt-format_cargo_toml: true` as a comment at the start of these files? that way we can test that `Cargo.toml` files aren't formatted when `format_cargo_toml=false`.

We can probably modify `read_significant_comments` to conditionally update the comment regex pattern to start with a `#` when we're dealing with a `Cargo.toml` file.

Reply:
Added